modules/sdk-evm/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@bitgo/sdk-evm",
+  "version": "1.0.0",
+  "description": "Bitgo SDK library for EVM Wallets",
+  "main": "./dist/src/index.js",
+  "type": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "yarn tsc --build --incremental --verbose .",
+    "fmt": "prettier --write .",
+    "check-fmt": "prettier --check .",
+    "clean": "rm -r ./dist",
+    "lint": "eslint --quiet .",
+    "prepare": "npm run build"
+  },
+  "dependencies": {
+    "@bitgo/sdk-api": "^1.9.5",
+    "@bitgo/sdk-core": "^5.0.0",
+    "@bitgo/sdk-test": "^1.2.10",
```

Comment:
sdk-test is a private package, should go in the `devDependencies`. `sdk-api` is only used by your tests, so it should also go in `devDependencies`